recipe.yaml

```diff
@@ -15,7 +15,7 @@ ComponentConfiguration:
   DefaultConfiguration:
     AutoProvision: 'true'
     InfluxDBMountPath: '/home/ggc_user/dashboard'
-    SecretArn: 'arn:aws:secretsmanager:<region>:<account>:secret:<name>'
+    SecretArn: 'arn:aws:secretsmanager:region:account:secret:name'
```

Comment:
why is this changed?

Reply:
To avoid potential parsing errors with the "<" ">" characters

Reply:
When is this a problem?

Reply:
I've seen it happen during the secret retrieval process- but this is just a placeholder anyway, and is expected to be replaced